### PR TITLE
Reformat Python part with black formatter

### PR DESCRIPTION
**Pull request by yurnov**
[Black](https://github.com/psf/black) is a PEP 8 compliant opinionated formatted for Python code widely used on Python Projects that follows [code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html).

This PR aimed to have easier readable unified from a style perspective Python code and includes a result of automated formatting of Python code with:
```
black --verbose --line-length 120 ./ecchronos-binary/src
```

Also black added to GitHub actions in the check mode to ensure that all following changes will keep the same codestyle.

No actual changes in the code besides formatting and GitHub Actions.

### CODE REVIEW

**Comment by yurnov** [comment]
Made `git commit --amend` and `git push --force` to [update the commit message](https://github.com/Ericsson/ecchronos/compare/5a59c81c53f0745224f0f8e9adf2654e1be44586..a1596703dc62f49f88c33fbb92569f1b597e3aac) as it had a typo.

File: ecchronos-binary/src/pylib/ecchronoslib/rest.py [hunk 1]
```diff
@@ -234,13 +228,19 @@ def post(self, keyspace=None, table=None, local=False, repair_type="vnode"):
...
             result = result.transform_with_data(new_data=[Repair(x) for x in result.data])
         return result
 
-    def get_repair_info(self, keyspace=None, table=None, since=None,  # pylint: disable=too-many-arguments, too-many-positional-arguments
-                        duration=None, local=False):
+    def get_repair_info(
+        self,
+        keyspace=None,
+        table=None,
+        since=None,  # pylint: disable=too-many-arguments, too-many-positional-arguments
```
Comment (tommystendahl) [inline ecchronos-binary/src/pylib/ecchronoslib/rest.py:240]: The comment suppressing pylint errors should be on L236

File: ecchronos-binary/src/bin/ecctool.py [hunk 2]
```diff
@@ -336,8 +445,12 @@ def get_jvm_opts(conf_dir):
 def run_ecc(cwd, command, arguments):
     if arguments.foreground:
         command += " -f"
-    proc = subprocess.Popen(command.split(" "), stdout=subprocess.PIPE, stderr=subprocess.STDOUT, # pylint: disable=consider-using-with
-                            cwd=cwd)
+    proc = subprocess.Popen(
+        command.split(" "),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,  # pylint: disable=consider-using-with
```
Comment (tommystendahl) [inline ecchronos-binary/src/bin/ecctool.py:451]: The comment suppressing pylint error should be on L448

**Review by tommystendahl — CHANGES_REQUESTED** [review]

**Comment by tommystendahl** [comment]
Hi @yurnov 
Thanks for your contribution, it looks really promising. Just two small pylint error, if you could update those it would be great.

File: ecchronos-binary/src/bin/ecctool.py (hunk 2 above)
Comment (yurnov) [reply ecchronos-binary/src/bin/ecctool.py:451]: Done

**Comment by yurnov** [comment]
Thanks @tommystendahl for the feedback, I updated PR

**Review by tommystendahl — APPROVED** [review]

**Comment by tommystendahl** [comment]
@yurnov Thanks for your contribution, looks good.

**Pull request merged**